### PR TITLE
Added a plugin override to change the way that resources display ther…

### PR DESCRIPTION
**Pull request by Roakz**
plugin extension for external resource locations including the link concatenation as per ticket 171802981

### CODE REVIEW

**Comment by Roakz** [comment]
@payten i guess i should do something similar here?

File: frontend/views/external_documents/_show.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <div class="subrecord-form-fields external-document">
+        <%= readonly.label_and_textfield "title" %>
+        <%= readonly.label_and_select "identifier_type", readonly.possible_options_for('identifier_type', true) %>
+
+        <% loc_uri = (document["location"] || "").slice(URI.regexp) %>
+        <% if loc_uri.nil? %>
+          <%= readonly.label_and_textfield "location" %>
+        <% else %>
+          <%= readonly.label_with_field("location", "#{document['location'].split('Link Address:')[0]}, 
+          Link Address: #{link_to(document["location"].split("Link Address:")[1], loc_uri)}") %>
```
Comment (payten) [inline frontend/views/external_documents/_show.html.erb]: This assumes that everything after the http is just the link.  However, `location` is a freetext field and could contain notes before/after this link.  

Also, `loc_uri` is only the link minus any gumpf if that's useful.

Comment (Roakz) [reply frontend/views/external_documents/_show.html.erb]: thanks mate ill check it out

**Comment by Roakz** [comment]
Let me know what you think @payten i had a crack

**Comment by payten** [comment]
That looks believable!

**Pull request merged**